src/devtools/client/debugger/src/actions/pause/selectFrame.ts

```diff
@@ -32,7 +33,16 @@ export function selectFrame(cx: Context, frame: TempFrame): UIThunkAction {
 
     dispatch(frameSelected({ cx, frameId: frame.id }));
 
-    await ThreadFront.getFrameSteps(frame.asyncIndex, frame.protocolId);
+    try {
+      await ThreadFront.getFrameSteps(frame.asyncIndex, frame.protocolId);
+    } catch (e: any) {
```

Comment:
is it better practice to do like `any | undefined | null` here and do a `e?.code` down below? I never know how to handle errors with typescript.

Reply:
Technically you can throw `null` in JavaScript, so it's probably _strictly_ best to let TypeScript tell you what the default type of `error` is and follow its rules.

Reply:
`unknown` is always preferable for types on bindings I think.

In my opinion the best way would be to declare a helper, e.g.
```
export function isProtocolError(e: unknown, code: ProtocolError): boolean {
  return e instanceof Error && e.name === "CommandError" && (e as any).code === code;
}
```
then do
```
} catch (e: unknown) {
  if (isProtocolError(e, ProtocolError.TooManyPoints)) {
```

Reply:
An error subclass would be an option too, but I sort of forget the state of extending the native Error type in browsers, I think it works?

Reply:
haha, I literally just wrote this code! Good, I'm glad we're in agreement!